spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -416,7 +410,9 @@ def add_electrical_series(recording, nwbfile, metadata):
         Auxiliary static method for nwbextractor.
         Adds traces from recording object as ElectricalSeries to nwbfile object.
         """
-        # ElectricalSeries
+        if 'Ecephys' not in metadata:
+            metadata['Ecephys'] = {}
+
         if 'ElectricalSeries' not in metadata['Ecephys']:
```

Comment:
```suggestion
        if 'Ecephys' not in metadata or 'ElectricalSeries' not in metadata['Ecephys']:
```

this can be used in place of `metadata['Ecephys'] = {}`

Reply:
Nice, I didn't know logically concatenated statements evaluated iteratively like that! Usually, it's the call to `metadata['Ecephys']` itself that triggers the issue of a missing key.

Reply:
yes, Python "or" statements have "short-circuit evaluation," so if the left side evaluates as True, the right side is never evaluated. Pretty handy in cases like this!